src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -31,6 +31,7 @@
 
     public static final String MESSAGE_SUCCESS = "New item added: %1$s";
     public static final String MESSAGE_DUPLICATE_ITEM = "This item already exists in the address book";
+    public static final String MESSAGE_ADDED_ON_EXISTING_ITEM = "Item added to stock. Stock is now: %1$s";
```

Comment:
Would item added to inventory be better?

Reply:
Yeap that sounds better! Made changes